src/esp/sim/ClassicReplayRenderer.cpp

```diff
@@ -31,6 +31,10 @@ ClassicReplayRenderer::ClassicReplayRenderer(
   resourceManager_ =
       std::make_unique<assets::ResourceManager>(metadataMediator, flags);
 
+  // hack to get ReplicCAD non-baked stages to render correctly
+  resourceManager_->getShaderManager().setFallback(
```

Comment:
Long-term, I hope @0mdc can figure out why the lighting isn't working correctly for the classic replay renderer.

Reply:
I remember I was doing some material patching for the ReplicaCAD composites, in particular [adding or removing the flat/unlit material bit](https://github.com/facebookresearch/habitat-sim/blob/07d0b3069068446c886cccb9fae8cffaf0783f2f/src/utils/compositor/compositor-replicacad.cpp#L230-L239) (not sure which was the case exactly), because otherwise both the prebaked and non-baked arch files were rendered with the same shader, looking wrong.

So if this works for you for the batch renderer & my composites but not the classic one, that might be the reason.